### PR TITLE
DOC-739: Added release notes for the media live embed enhancement

### PR DESCRIPTION
**Pull request by lnewson**
Related Ticket: DOC-739

Description of Changes:
* Adds a release note for the new audio/video live embeds enhancement. I didn't update the option, as it was already vague and never mentioned it only worked for iframes previously.

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

**Review by JamesToohey — APPROVED** [review]

**Review by ashfordneil — APPROVED** [review]

File: release-notes/release-notes57.md [hunk 1]
```diff
@@ -39,6 +39,12 @@ The following enhancements were made for the {{site.productname}} 5.7 release.
 
 ### Enhancement name
 
+### Media live embeds now support video and audio elements
```
Comment (metricjs) [inline release-notes/release-notes57.md:42]:
```suggestion
### Media live embeds now supports video and audio elements
```
maybe? just read weirdly to me

**Review by metricjs — APPROVED** [review]

File: release-notes/release-notes57.md (hunk 1 above)
Comment (lnewson) [reply release-notes/release-notes57.md:42]: Will leave that call up to @tylerkelly13, as I prefer `support` here.

Comment (tylerkelly13) [reply release-notes/release-notes57.md:42]: Maybe:
> The Media Live Embeds feature now supports video and audio elements

?

**Review by tylerkelly13 — APPROVED** [review]

File: release-notes/release-notes57.md (hunk 1 above)
Comment (metricjs) [reply release-notes/release-notes57.md:42]: Not sure `feature` makes sense in this context...

Comment (tylerkelly13) [reply release-notes/release-notes57.md:42]: depends on how you read it. Media Live Embeds could be referring to the feature overall, the option, or the embedded element created using the feature + option.

Comment (tylerkelly13) [reply release-notes/release-notes57.md:42]: Leaving as-is

**Review by EkimChau — APPROVED** [review]

**Pull request merged**